dot/state/interfaces.go

```diff
@@ -29,10 +29,11 @@ type GetPutter interface {
 	Putter
 }
 
-// GetNewBatcher has methods to get values and create a
+// Batcher has methods to get values and create a
 // new batch.
-type GetNewBatcher interface {
+type Batcher interface {
```

Comment:
`Batcher` highlights only one method of this interface (`NewBatch`).
I think it is common to name interface back on their methods, when we don't have other better names that would fit them.
So, names `GetterPutter` or `GetterPutterNewBatcher`, etc are acceptable.